.gitignore

```diff
@@ -6,6 +6,7 @@
 *.db
 *.log
 /*.env
+.idea/
```

Comment:
This is the kind of thing you should probably put in your local configuration rather than on a project level. It's better if you put it in your global gitignore anyways since this means you won't have to have this in all your projects. ;)